pkg/authorization/api/types.go

```diff
@@ -20,18 +20,53 @@ const (
 	VerbAll     = "*"
 )
 
+const (
+	ResourceGroupPrefix         = "resourcegroup"
+	BuildGroupName              = ResourceGroupPrefix + ":builds"
+	DeploymentGroupName         = ResourceGroupPrefix + ":deployments"
+	ImageGroupName              = ResourceGroupPrefix + ":images"
+	OAuthGroupName              = ResourceGroupPrefix + ":oauth"
+	UserGroupName               = ResourceGroupPrefix + ":users"
+	PolicyOwnerGroupName        = ResourceGroupPrefix + ":policy"
+	PermissionGrantingGroupName = ResourceGroupPrefix + ":granter"
+	OpenshiftExposedGroupName   = ResourceGroupPrefix + ":exposedopenshift"
```

Comment:
> is there a clear line between exposed and private openshift and kube resources?

There isn't a particularly clear line, but some are intended to be system managed and user visible and others are intended to be user managed.  For instance, most users will never have a reason to create an endpoint or an event.